### ISSUE TITLE
Explicitly add Log files to View intent

### ISSUE THREAD

**Issue by debugrr** [termux/termux-app#1497]
**Feature description**

Proposing adding the following line to the code block in the manifest. 
```
     <data android:mimeType="application/*log*" />
```
Some applications do not respect text/*.

```
            <!-- Be more restrictive for viewing files, restricting ourselves to text files. -->
            <intent-filter tools:ignore="AppLinkUrlError">
                <action android:name="android.intent.action.VIEW"/>
                <category android:name="android.intent.category.DEFAULT" />
                <data android:mimeType="text/*" />
                <data android:mimeType="application/json" />
                <data android:mimeType="application/*xml*" />
                <data android:mimeType="application/*latex*" />
                <data android:mimeType="application/javascript" />
            </intent-filter>
```
**Reference implementation**

Does another app/terminal emulator have this feature?
Provide links to more background information.

Want to be about to pass a log file from an email to vim inside termux. 
Current implementation Android doesn't recognize termux as a valid app for .log files. 

And this is not necessarily a bug for Termux, current code should allow this.

**Comment by mwalimu-prog** [comment]
Ok. Thanks

**Comment by xeffyr** [comment]
Fixed in v0.95.
